web/src/routes/TuNewsPage.tsx

```diff
@@ -74,13 +76,18 @@ const TuNewsPage = ({ cityCode, languageCode, cityModel, languages }: CityRouteP
     )
   }
 
+  const toolbar = (openFeedback: (rating: FeedbackRatingType) => void) => (
```

Comment:
A little weird to put a toolbar below the TuNews List in mobile as it is growing indefinitely. I'll never reach the damn copy button. Maybe we omit it for the small viewport?

Reply:
yes thats currently the agreement. i can disabled it for mobile viewport